### PR TITLE
Save all pictures button in admin/cards

### LINKED ISSUE #420: Add save all button in card generation (CanvasContainer.vue)
## Description
Currently, there is only the ability to save on image from the card generation. It should also be possible to save all at once in corresponding image folders. 

## Acceptance criteria
- [ ] Functionality to save all images at once from card generation.

### PR DESCRIPTION
**Pull request by te4-dennis-christensen**
## ⚙️ What issue did you implement or fix?
Fixes #420

## 📜 Summary
- Added a button to save all pictures

## 📝 Type of change
- [x] ✨ New feature or enhancement (non-breaking change which adds functionality)

## ✅ Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

## 💩 How can this implementation be improved?
Don't know how, just doesn't feel like the best solution.

### CODE REVIEW

**Review by te4-adrian-almetun-smeds — CHANGES_REQUESTED** [review]
Looks good, I'd like one change though.

And yes the better solution would probably be to setup a rails route that accepts multiple updates at the same time, but that also sort of violates the RESTful API architecture. So there's both good and bad views on the current implementation.

File: frontend/src/components/CanvasContainer.vue [hunk 1]
```diff
@@ -39,8 +43,9 @@ export default defineComponent({
   props: {
     images: {type: Array, default: []}
   },
-  setup(props : CanvasContainerProps){
+  setup(props : CanvasContainerProps, { root }: SetupContext){
     let sendCanvas: Ref<boolean> = ref(false);
+    let saveAllPictures: Ref<boolean> = ref(false);
```
[the remark this answers is not in the document]
Comment (te4-dennis-christensen) [reply frontend/src/components/CanvasContainer.vue]: fixed in latest commit

**Review by te4-adrian-almetun-smeds — APPROVED** [review]
I approve :+1:

**Review by te4-andre-skvarc — APPROVED** [review]
Looks good!

**Pull request merged**